Client/tests/dbsclient_t/utils/DBSDataProvider.py

```diff
@@ -1,4 +1,4 @@
-import cPickle as pickle
+import pickle as pickle
```

Comment:
It won't make any difference, but it could be just `import pickle`

Reply:
agreed